apps/wasm/mina-berkeley/static/manifest.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
Files under static are for chrome extension v3, I don't think they're needed in this case.